Pipfile

```diff
@@ -80,11 +78,13 @@ django-filter = "*"
 click = "*"
 flask = "==2.3.2"
 certifi = "*"
+Markdown = "*"
 
```

Comment:
can you add a 

```
[requires]
python_version = "3.12"
```

in the main `Pipfile` so that when I create an env locally it creates the env with the right python version without me having to specify it manually.

Reply:
Fixed with https://github.com/DemocracyClub/electionleaflets/pull/255/commits/20fa7d967e09a077f6e4cf732c86182879def80d.